src/common/offline_transformations/include/mask_attribute.hpp

```diff
@@ -180,6 +180,26 @@ class Mask : public std::vector<std::set<uint64_t>>, public std::enable_shared_f
         return result_mask;
     }
 
+    void copy_and_slice_mask_from(const ngraph::Mask* const mask,
+                                  int64_t axis,
+                                  uint64_t split_start,
+                                  uint64_t split_end) {
```

Comment:
Please add doc for this method

Reply:
done

Reply:
Could you please format the comment as other `Mask` method comments formatted? It would be great!

Reply:
done